src/main/java/apoc/periodic/Periodic.java

```diff
@@ -49,11 +50,15 @@
 
     @Procedure(mode = Mode.WRITE)
     @Description("apoc.periodic.commit(statement,params) - runs the given statement in separate transactions until it returns 0")
```

Comment:
should we document that it needs a `LIMIT $limit` ?

Reply:
Agreed. Will fix this.